BoxContentSDK/BoxContentSDK/Requests/BOXCommentDeleteRequest.h

```diff
@@ -10,6 +10,8 @@
 @property (nonatomic, readwrite, strong) NSURL *sharedLinkURL;
 @property (nonatomic, readwrite, strong) NSString *sharedLinkPassword; // Only required if the shared link is password-protected
 
+@property (nonatomic, readonly, strong) NSString *commentID;
```

Comment:
This will provide an easy way for whoever implements the cache to find out which comment is about to be deleted.